packages/truffle-workflow-compile/test/index.js

```diff
@@ -25,30 +36,28 @@ describe("Contracts.compileSources", () => {
 });
 
 describe("Contracts.compile", () => {
-  it("when config.all is true, all contracts in contracts_directory are recompiled", () => {
+  it("when config.all is true, all contracts in contracts_directory are recompiled", async () => {
     let contractName;
 
     // initial compile
-    Contracts.compile(config, (err, { contracts }) => {
+    await Contracts.compile(config, (err, { contracts }) => {
       if (err) assert.fail(err);
-      else {
-        contractName = Object.keys(contracts)[0];
-        assert(
-          existsSync(`${config.contracts_build_directory}/${contractName}.json`)
-        );
-      }
+      contractName = Object.keys(contracts)[0];
+      assert(
+        existsSync(`${config.contracts_build_directory}/${contractName}.json`)
+      );
     });
 
     // compile again
     config.all = true;
-    Contracts.compile(config, (err, output) => {
+    await Contracts.compile(config, (err, { outputs }) => {
       if (err) assert.fail(err);
-      else {
-        assert(
-          output.solc ===
+      assert(
+        outputs.solc[0] ===
+          join(
             `${process.cwd()}/${config.contracts_directory}/${contractName}.sol`
-        );
-      }
+          )
+      );
     });
-  });
+  }).timeout(3000);
```

Comment:
Was this failing without the added timeout? Seemed to be passing on my local yesterday.

Reply:
Same on my local. Travis is slowww.